aptos-move/framework/aptos-token/sources/token_coin_swap.move

```diff
@@ -33,6 +33,7 @@ module aptos_token::token_coin_swap {
         listings: Table<TokenId, TokenCoinSwap<CoinType>>,
         listing_events: EventHandle<TokenListingEvent>,
         swap_events: EventHandle<TokenSwapEvent>,
```

Comment:
same comment as below, we need a new resource to contain the events otherwise we make a breaking ABI incompatible change